testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -122,9 +122,15 @@ protected QueryRunner createQueryRunner()
             public Iterable<ConnectorFactory> getConnectorFactories()
             {
                 MockConnectorFactory connectorFactory = MockConnectorFactory.builder()
-                        .withListTables((session, s) -> ImmutableList.of(
-                                new SchemaTableName("default", "tests_table"),
-                                new SchemaTableName("tiny", "nation")))
+                        .withListTables((session, schemaName) -> {
+                            if (schemaName.equals("default")) {
```

Comment:
nit: In some methods is used `equalsIgnoreCase` and some methods use `equals`.
It would be good to have this kind of checks consistent across the code base.

Reply:
I agree. I think i didn't introduce any new equalsIgnoreCase and didn't remove any existing equalsIgnoreCase occurrences though.
This would be suboptimal to mix two different refactors together.